src/comps/update/index.js

```diff
@@ -23,21 +19,19 @@ async function updateDb(data, dbname, round) {
             values.push(round)
         }
 
-        let query = `INSERT INTO ${dbname} (${keys.join(", ")}) VALUES (${values
-            .map((x) => {
-                if (isNaN(x) || x.toString().startsWith("0x")) {
-                    let str = `${x}`
-                    str = str.replace(/%@#/g, ",")
-                    str = db.escape(str)
-                    return str
-                } else {
-                    return parseFloat(x)
-                }
-            })
-            .join(", ")})`
-
-        let res = await db.promise().query(query)
-    })
+        // Create placeholders for parameterized query
+        const placeholders = keys.map(() => "?").join(", ")
```

Comment:
Awesome update to make it more easily verifiable in tests.
Reviewing test coverage.